The origin error:
error[E0277]: a value of type `String` cannot be built from an iterator over elements of type `u8`
    --> src/main.rs:9:56
     |
9    |         *p = (0..5).map(|_| rng.sample(&Alphanumeric)).collect()
     |                                                        ^^^^^^^ value of type `String` cannot be built from `std::iter::Iterator<Item=u8>`
     |
     = help: the trait `FromIterator<u8>` is not implemented for `String`

--- a/src/concurrency/parallel/rayon-parallel-sort.md
+++ b/src/concurrency/parallel/rayon-parallel-sort.md
@@ -22,7 +22,7 @@ fn main() {
   let mut vec = vec![String::new(); 100_000];
   vec.par_iter_mut().for_each(|p| {
     let mut rng = thread_rng();
-    *p = (0..5).map(|_| char::from(rng.sample(&Alphanumeric))).collect()
+    *p = (0..5).map(|_| rng.sample(&Alphanumeric) as char).collect()
   });
   vec.par_sort_unstable();
 }

